web/beacon-app/src/features/auth/routes/LoginPage.tsx

```diff
@@ -12,6 +12,24 @@ import LoginForm from '../components/Login/LoginForm';
 import { useLogin } from '../hooks/useLogin';
 import { isAuthenticated } from '../types/LoginService';
 
+const StyledButton = styled(Button)((props) => ({
```

Comment:
This was the best way for me to get the ghost variant working properly until changes are made to it in the design system.